scripts/pl-install.sh

```diff
@@ -79,7 +79,7 @@ curl -LO https://github.com/conda-forge/miniforge/releases/download/24.3.0-0/Min
 bash Miniforge3-Linux-${arch}.sh -b -p /usr/local -f
 
 echo "installing Python packages..."
-python3 -m pip install --no-cache-dir -r /python-requirements.txt
+python3 -m pip install --no-cache-dir -r /requirements.txt
```

Comment:
👀 now that we don't have the `plbase` split, does `/requirements.txt` still need to live at the root? Why not copy it to `/PrairieLearn/requirements.txt` like we do with, well, the rest of the files?

Reply:
Fixed in [0ce52bf](https://github.com/PrairieLearn/PrairieLearn/pull/12293/commits/0ce52bf09914e9455bd038e43f70b1c9c6c02cb4).

Reply:
actually 046d541a3bd52c74a2b3d4733ebcde722baa3823